docs/http-api/projections.md

```diff
@@ -221,6 +235,8 @@ The server then returns the state for the partition:
 * `deleteCheckpointStream`: Delete the checkpoint stream (true/false)
```

Comment:
Should the arguments be shown in the example?

Reply:
from the other examples, it looks like just including it in the URI is setting it true. Omitting is false. This is unclear in the documentation though. If that's the case, I think we should just remove the (true/false) from the descriptions. So, it's also unclear if any/all of these parameters are required.

Reply:
> it looks like just including it in the URI is setting it true. Omitting is false.

@rsowald This isn't always the case. Omitting it is leaving it as the default (which we should list in the parameters), but if you want to set it you should use either `deleteCheckpointStream=true` or `deleteCheckpointStream=false`

> Should the arguments be shown in the example?

@cdevarenne Do you mean something like this?

```
/projections/continuous?name={name}&type={ js | native }&enabled={ true | false }&emit={ true | false }&trackemittedstreams={ true | false }
```